cypress/integration/05.ReportContent.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Report and Moderate
+  As a user
+  I would like to report content that viloates the community guidlines
+  So the moderators can take action on it
+
+  As a moderator
+  I would like to see all reported content
+  So I can look into it and decide what to do
+
+  Scenario: Report a Post from landingpage
+    Given I am logged in as "user"
+    And I am on the "landing" page
+
+    When I click on a Post menu and select the report option
+    And I click on send in the confirmation dialog
+    Then I get a success message
+
+  Scenario: See reported content
+    Given I am logged in as "moderator"
+    And I previously reported a post
+
+    When I am on the "moderation" page
+    Then I see my reported post
+
+  Scenario: Report while reading Post
+    Given I am logged in as "admin"
+    And I am viewing a post
+
+    When I report the current post
+    And I visit the "moderation" page
+    Then I see my reported post
+
+  Scenario: Report user
+    Given I am logged in as "admin"
+    And I am viewing a post
+
+    When I click on the author
+    And I report the author
+    And I visit the "moderation" page
+    Then I see my reported user
+
+  Scenario: Normal user can't see the moderation page
+    Given I am logged in as "user"
+
+    When I can click on my profile picture in the top right corner
+    Then I can't see the moderation menu item
```

Comment:
The absence of a UI is really hard to test. I would suggest to remove that scenario. Only test what is there, not what is not there.

From the implementation it's hard to check for non-existence, e.g.checking a blank page for text not being there is always `true`.